docs/tutorials/further_topics/approximate_moran_processes.rst

```diff
@@ -26,8 +26,9 @@ Now let us create an Approximate Moran Process::
     >>> amp.population_distribution()
     Counter({'Random: 0.5': 3})
 
-We see that the :code:`Random: 0.5` won this Moran process. This is not what happens in a
-standard Moran process where the `Random: 0.5` player will not win::
+We see that the :code:`Random: 0.5` won this Moran process. This is not what
```

Comment:
Let's clarify that this happens only for a particular choice of random seed -- sometimes Random will win.